lib/active_merchant/billing/gateways/vpos.rb

```diff
@@ -146,15 +181,21 @@ def success_from(response)
       end
 
       def message_from(response)
-        response.dig('confirmation', 'extended_response_description') ||
-          response.dig('confirmation', 'response_description') ||
-          response.dig('confirmation', 'response_details') ||
-          response.dig('messages', 0, 'key')
+        %w(confirmation refund).each do |m|
+          message =
+            response.dig(m, 'extended_response_description') ||
+            response.dig(m, 'response_description') ||
```

Comment:
Does the `||` at the end of the above 2 lines act the same way as when they are used for things all on the same line? I like the break up here, nice and easy to read!

Reply:
It does! (In fact, I think Rubocop might fuss at you for putting multiple `||` on the same line?)